src/app/public/src/modules/page-anchor/page-anchor.component.ts

```diff
@@ -19,7 +19,7 @@ export class StachePageAnchorComponent implements OnInit, StacheNavLink {
     private elementRef: ElementRef,
     private windowRef: StacheWindowRef) { }
 
-  public ngOnInit(): void {
+  public ngAfterContentChecked(): void {
```

Comment:
This needed to be changed in order to be able to use variables in the page anchor component. With the way it was before using `ngOnInit`, the content would render, but the TOC would be blank. It's a bug that Lindsey filed a while back.